dietpi/func/dietpi-set_core_environment

```diff
@@ -279,60 +279,14 @@ exit 0
 _EOF_
 	chmod +x /var/lib/dietpi/dietpi-software/services/kill-ssh-user-sessions-before-network.sh
 
-	G_DIETPI-NOTIFY 2 'Configuring rc-local service:'
```

Comment:
This needs to be added to PREP?

Reply:
Or we could: keep the service creation in core-env, then manually create ```/etc/rc.local``` outside of it?

Reply:
Do we want to assure rc.local functionality with DietPi? By default it is implemented moreless, if I remember right rc.local file itself is not present on all Debian images by default, but the services are in place.

I thought about not touching this on new images, leaving it to user if needed and just assuring that it runs after DietPi-PostBoot (Before=).

Reply:
@MichaIng 

> I thought about not touching this on new images, leaving it to user if needed and just assuring that it runs after DietPi-PostBoot (Before=).

Agree, postboot is a better feature/system 👍  Lets go with that and leave rc-local as is on new images.